pythonPackages.pysonos: init at 0.0.11

oneingan opened the pull request:
pysonos package provides ability to control sonos media player using python. The original need was to use it with the home-assistant package, specifically the "sonos" component.

fgaz committed:
twemoji-color-font: 11.2.0 -> 12.0.1

9ary committed:
nginx: use fullchain.pem for ssl_trusted_certificate

Some ACME clients do not generate full.pem, which is the same as
fullchain.pem + the certificate key (key.pem), which is not necessary
for verifying OCSP staples.

azazel75 committed:
grafana_reporter: Fix library function name

Commit:
pythonPackages.pysonos: init at 0.0.11

wucke13 committed:
s-tar: 1.5.3 -> 1.6

etu committed:
emby: Drop package and module and refer to jellyfin

jtojnar committed:
gnomeExtensions.gsconnect: 21 → 23

https://github.com/andyholmes/gnome-shell-extension-gsconnect/releases/tag/v22

Commit:
Merge remote-tracking branch 'upstream/master'

r-ryantm committed:
python37Packages.pyatspi: 2.32.0 -> 2.32.1

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/python3.7-pyatspi/versions

fgaz committed:
twemoji-color-font: automatically construct "name"

jtojnar committed:
dropbox-cli: Fix and add Nautilus extension

…since this is actually dropbox-nautilus project.

In one of the previous updates, the source code responsible for finding dropboxd
was changed, rendering our sed replacement not working. That triggered the installer
which in turn failed on the unavailability of gobject-introspection typelibs. And
when that was fixed, we got bitten by our lazy packaging putting a broken code
into the installer.

This is a proper fix for all the issues, except for making typelibs available to
the installer since we shall use Nix instead.

xrelkd committed:
rustup: 1.18.1 -> 1.18.2

pacien committed:
riot-web: add pacien as maintainer

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
parse-requirement.py generated on upstream merge

r-ryantm committed:
translate-shell: 0.9.6.9 -> 0.9.6.10

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/translate-shell/versions

JohnAZoidberg commented on pkgs/development/python-modules/pysonos/default.nix:
```diff
@@ -0,0 +1,28 @@
+
+buildPythonPackage rec {
+  pname = "pysonos";
+  version = "0.0.11";
```
0.0.12 is out, please upgrade.

JohnAZoidberg commented:
Why did you remove the standard checkboxes from the PR message?

Please squash the commits. Hint: Use rebase instead of merge when you want to update your private branch.

r-ryantm committed:
typora: 0.9.68 -> 0.9.70

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/typora/versions

pacien committed:
riot-desktop: 1.0.7 -> 1.0.8

pacien committed:
riot: add maintainer note

r-ryantm committed:
vnstat: 2.1 -> 2.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/vnstat/versions

thoughtpolice committed:
liburing: bump, fix version (1.0.0pre92_7b989f3)

I accidentally got the number of commits wrong in the previous
prerelease version string. This is now fixed.

Generally, this would result in functions like builtins.compareVersions
to give incorrect results, so 'nix-env -u' doesn't work. But I'm
justifying it here, because: most people use it as a library, so the
hash change is all that matters. Plus, I only authored this a week or
so ago in upstream, so this change is fast enough that I think people
will be fine with it and can work around, especially since it's
unreleased in any stable channel.

This also bumps the library to the newest version, which contains some
bugfixes, and now installs the manpages into the $man output for us.

Signed-off-by: Austin Seipp <aseipp@pobox.com>

jtojnar committed:
common-updater-scripts: fix fallback version detection

parseDrvName returns a set containing version attribute, not our customizable key.

jtojnar committed:
sublime3-dev: 3203 → 3208

Build 3208
21 April 2019

    Mac: Added a workaround for a macOS OpenGL driver bug in 10.14.4

Build 3206
5 April 2019

    Performance improvements

Build 3205
3 April 2019

    Improved scrolling logic in some scenarios
    Linux: Fixed compatibility with old Linux distributions

Build 3204
2 April 2019

    Mac: Added a workaround for a MacOS issue with DisplayLink adapters
    Linux: Tweaked the way text scaling is handled
    Improved file indexing behavior in some scenarios
    Fixed block carets changing the way text selection works

juaningan commented:
Please delete this. I mess it.

juaningan commented:
new attempt in #60853
